include/psen_scan_v2/scanner_state_machine.h

```diff
@@ -118,6 +118,8 @@ class ScannerProtocolDef : public msm::front::state_machine_def<ScannerProtocolD
 public:  // Action methods
   template <class T>
   void sendStartRequest(const T& event);
+  template <class T>
+  void handleStartRequestTimeout(const T& event);
```

Comment:
Why is this action a template function? It does not have to be because this action is only called with one event (`StartTimeout`).

Reply:
Fixed -> Done.